apps/workspaces/serializers.py

```
@@ -143,10 +136,56 @@ def create(self, validated_data):
             workspace.onboarding_state = 'EXPORT_SETTINGS'
             workspace.save()
 
+            return sage300_credentials
+
+        except Exception as e:
+            logger.info(e)
+            raise serializers.ValidationError('Invalid Login Attempt')
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)